client/components/SongBlock/index.module.styl

```diff
@@ -22,11 +19,5 @@
         clear both
         float left
         color inherit
-        &:hover, &:focus
-          color linkHoverColor
   &.playing
     color currentlyPlayingColor
-  &:focus
```

Comment:
These focus's a lot of the time are for accessibility reasons, for someone to be able to tab through the UI

Reply:
Absolutely, I still need to restore a lot of these